frontend/src/wrapper/NavMenuItem.tsx

```diff
@@ -1,7 +1,6 @@
 import { Badge, ListItem, ListItemButton, ListItemIcon, ListItemText, Tooltip } from '@mui/material'
 import { useTheme } from '@mui/material/styles'
 import { ReactElement } from 'react'
-import Link from 'src/Link'
```

Comment:
You probably don't want to disable tabbing down the sidebar.  It's important that users without a mouse are still able to navigate.